CME/dataobjects/CMEProvider.php

```diff
@@ -67,6 +68,27 @@ public function loadByShortname($shortname)
 	}
 
 	// }}}
+	// {{{ public function getCreditTitle()
+
+	public function getCreditTitle($hours, $credit_count = 1, $is_free = false)
+	{
+		$locale = SwatI18NLocale::get();
+		return sprintf(
+			SwatString::minimizeEntities(
+				CME::_('%s%s %s%s%s certified by %s')
```

Comment:
If we do keep this as one translatable string ([see discussion](https://github.com/silverorange/Cme/pull/58/files#r32939747)), there needs to be a space between the first replacement marker and the second.